tests/core/adjust/data/tests.fmf

```diff
@@ -0,0 +1,15 @@
+/free:
+    test: free
+
+/uptime:
+    test: uptime
+    adjust:
+        duration: 1m
+        when: arch == ppc64
+
+/pidof:
+    test: pidof sh
+    adjust:
+        enabled: false
+        when: distro < centos-8
+        because: pidof is available in centos-8 and later
```

Comment:
Could be written the other way around as well, right?

```
enabled: false
adjust:
    enabled: true
    when:  distro >= centos-8 or distro >= fedora-22    # guessing, feel free to correct me
```

Reply:
It should be `distro ~>= centos-8 or distro ~>= fedora-22`

Reply:
Yes, both ways should work.

Reply:
Thanks!